bitnami/shared/upstream/kustomization.yaml

```diff
@@ -38,3 +38,14 @@ patches:
     target:
       group: apps
       version: v1 # Deployment | StatefulSet
+  # remove these empty affinity nodes, as in Kustomize 5.0.x kustomize generates them as strings with value "null" instead of just null, and they can not be applied.
```

Comment:
Is this a bug, is it an issue just with this `kustomization`? (do we need to file a report somewhere/does one already exist?)

Reply:
:point_up: I vote for raising a GH issue with them

Reply:
yea, I wanted to do that after we finish with this. 
It is tricky to reproduce though, as it happens only if you build from kube-manifests now. 
If you build it here, it will output null, not as a string, which is ok.